Client/Frontend/Settings/SearchCustomEngineViewController.swift

```diff
@@ -0,0 +1,727 @@
+// Copyright 2020 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import Foundation
+import Static
+import Shared
+import WebKit
+import SnapKit
+import Fuzi
+import Storage
+import Data
+
+private let log = Logger.browserLogger
+
+// MARK: - SearchCustomEngineViewController
```

Comment:
nit: Mark's for class definitions are not needed as they are visible from the file navigator in Xcode